test-e2e/har/har.ts

```diff
@@ -0,0 +1,120 @@
+import * as _ from 'lodash';
```

Comment:
`_` is usually used to refer to an unused variable in typescript

does `import lodash from 'lodash'` works?
even better, how about importing only the functions we're using with `import { foo, bar } from 'lodash'` ?